src/components/Notification/Notification.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+import { Transition } from 'react-transition-group'
+
+import { Button, Icon } from '../../../src'
+
+const propTypes = {
+  item: {
+    show: PropTypes.bool,
+    key: PropTypes.string,
+    type: PropTypes.string,
+    position: PropTypes.string,
+    isDismissable: PropTypes.bool,
+    isWithIcon: PropTypes.bool,
+    icon: PropTypes.string,
+    title: PropTypes.string,
+    text: PropTypes.string
+  },
+  close: PropTypes.func
+}
+
+class Notification extends React.Component {
```

Comment:
I do not see any particular need to use a class here, a Functional Component should be enough.